config/routes.rb

```diff
@@ -50,6 +50,8 @@
 
   namespace :changelog_admin do
     resources :entries, only: [:new, :create, :show, :index]
+
+    root to: "entries#index"
```

Comment:
Didn't know you could do this. This is cool. I guess the only downside is that we end up with two URLs with the same content? (`/changelog_admin` and `/changelog_admin/entries`)? Any downsides worth worrying about with that?

Reply:
I think it's okay for now. I just wanted to route to the entries index when someone goes to `/changelog_admin` rather than getting a 404 error.

In the future, having a dashboard controller which routes to root seems the way to go.